src/sentry/testutils/factories.py

```diff
@@ -255,6 +256,14 @@ def create_organization(name=None, owner=None, **kwargs):
             Factories.create_member(organization=org, user=owner, role="owner")
         return org
 
+    @staticmethod
+    @exempt_from_silo_limits()
+    def create_organization_mapping(org, **kwargs):
+        slug = kwargs.get("slug", org.slug)
+        kwargs["slug"] = slug
```

Comment:
You could use `kwargs.setdefault("slug", org.slug)` to set the key if it doesn't already exist.